src/Features/LanguageServer/Protocol/Workspaces/LspWorkspaceManager.cs

```diff
@@ -160,15 +157,15 @@ public async Task<(Workspace?, Solution?)> GetLspSolutionInfoAsync(CancellationT
         // Ensure we have the latest lsp solutions
         var updatedSolutions = await GetLspSolutionsAsync(cancellationToken).ConfigureAwait(false);
 
-        var (hostWorkspace, hostWorkspaceSolution, isForked) = updatedSolutions.FirstOrDefault(lspSolution => lspSolution.Solution.WorkspaceKind == _hostWorkspaceKind);
+        var (hostWorkspace, hostWorkspaceSolution, isForked) = updatedSolutions.FirstOrDefault(lspSolution => lspSolution.Solution.WorkspaceKind == WorkspaceKind.Host);
```

Comment:
So the fact that all the methods returned WorkspaceKind.Host is _somewhat_ a coincidence, in that I could imagine at some point the main workspace wouldn't have that kind. At least, if ever we find ourselves in that situation, I'd expect rather than changing the workspace kind we'd revert this PR.

Reply:
Agreed.